src/Continuous_Learning/continuous-learning-get-and-submit.sh

```diff
@@ -0,0 +1,38 @@
+#! /bin/bash
+
+set -e
+set -x
+
+export DATE=`date +"%d%m%Y-%H%M"`
+# export HOME=/pfs/nobackup$HOME
+
+OpenNMT_py="$HOME/OpenNMT-py"
+CONTINUOUS_LEARNING_PATH="$HOME/sequencer/src/Continuous_Learning"
+DEFECTS4J_EXPERMENT_PATH="$HOME/sequencer/src/Defects4J_Experiment"
+
+export DATA_PATH="$CONTINUOUS_LEARNING_PATH/processed/$DATE"
+VOCABULARY="$HOME/sequencer/results/Golden/vocab.txt"
```

Comment:
I think it is better to update the vocab file.

For example if you are training a new model, the vocabulary can differ a lot to the golden model